Gaijinizer patch/Map085.txt

```diff
@@ -168,40 +172,41 @@ was really popular.#MTLed
 最初は緊張でトギマギしちゃってたけど、
 だんだん歌も踊りも、司会も慣れていったんだ。
 > CONTEXT: Map085/events/5/pages/1/38/Dialogue/enemy6
-I'm pleased with cheers and applause, view rate
-... At first I was nervous because of tension,
-Once I got used to singing and dancing, and the
-presidency.#MTLed
+The cheers, applause, and ratings made me really
+happy... I was really nervous and overwhelmed at
+first, but I gradually got used to singing,
+dancing, and presenting.#RemovedTranslation
```

Comment:
This is the dialogue that needs its translation re-added.

Reply:
Looks like there's no pointer change on this one, so it was an easy fix. Done.